packages/manager/modules/pci/src/projects/project/kubernetes/add/add.html

```diff
@@ -10,7 +10,7 @@ <h1 data-translate="kubernetes_add"></h1>
         data-valid="$ctrl.cluster.region"
         data-navigation="$ctrl.cluster.region"
         data-on-focus="$ctrl.displaySelectedRegion = false"
-        data-on-submit="$ctrl.displaySelectedRegion = true"
+        data-on-submit="$ctrl.displaySelectedRegion = true; $ctrl.onRegionSubmit()"
```

Comment:
Maybe `$ctrl.displaySelectedRegion = true;` could be moved to the `onRegionSubmit()`